src/init.cpp

```diff
@@ -577,7 +577,8 @@ void SetupServerArgs(NodeContext& node)
     argsman.AddArg("-server", "Accept command line and JSON-RPC commands", ArgsManager::ALLOW_ANY, OptionsCategory::RPC);
 
 #if HAVE_DECL_DAEMON
-    argsman.AddArg("-daemon", "Run in the background as a daemon and accept commands", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-daemon", "Run in the background as a daemon and accept commands (default: 0)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-daemonwait", "Wait for initialization to be finished before exiting. This implies -daemon (default: 0)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
s/`ALLOW_ANY`/`ALLOW_BOOL`/ in both lines?

Reply:
Good idea!
I'm also surprised there are no DEFAULT_XXX constants here.